tests/openuri.c

```diff
@@ -478,3 +479,59 @@ test_open_directory (void)
   while (!got_info)
     g_main_context_iteration (NULL, TRUE);
 }
+
+void
+test_scheme_supported(void) 
+{
+  g_autoptr(GError) error = NULL;
+  g_autoptr(GVariant) result = NULL;
+  g_autoptr(GDBusConnection) session_bus = g_bus_get_sync (G_BUS_TYPE_SESSION, NULL, &error);
+
+  XdpDbusOpenURI *proxy = xdp_dbus_open_uri_proxy_new_sync (session_bus,
```

Comment:
```suggestion
  g_autoptr(XdpDbusOpenURI) proxy = xdp_dbus_open_uri_proxy_new_sync (session_bus,
```

Reply:
Hm, looks like it does not work fine:
```
../tests/openuri.c: In function ‘test_scheme_supported’:
../tests/openuri.c:493:13: error: unknown type name ‘XdpDbusOpenURI_autoptr’; did you mean ‘XdpDbusOpenURIProxy’?
  493 |   g_autoptr(XdpDbusOpenURI) proxy = xdp_dbus_open_uri_proxy_new_sync (session_bus,
      |             ^~~~~~~~~~~~~~
../tests/openuri.c:493:3: error: cleanup argument not a function
  493 |   g_autoptr(XdpDbusOpenURI) proxy = xdp_dbus_open_uri_proxy_new_sync (session_bus,
      |   ^~~~~~~~~
```
Looks like XdpDbusOpenURI is not g_autoptr compatible.

Reply:
Huh, indeed. We turn off autocleanup for most GDbus generated sources. I'll try to figure out if we can enable it but for this PR manually unref'ing is okay then.

Reply:
When https://github.com/flatpak/xdg-desktop-portal/pull/1468 gets merged this shouldn't be an issue anymore.

Reply:
It has already been merged so you could rebase and update the code to use g_autoptr again.